README.rst

```diff
@@ -340,9 +336,10 @@ like 'radosgw-agent'. This inclusion would be defined at the project level, like
     repos = {
         'ceph': {
             'extras': ['ceph-deploy', 'radosgw-agent'],
-            'deb': {
```

Comment:
we should default to something, like `multi`, `combined`, or `mixed`. I think having one of these will be better than to rely on always-correct configuration.